### ISSUE TITLE
Add subtitles

### ISSUE THREAD

**Issue by terry-dactle** [d4rken/fpv-dvca#5]
Love the app, any chance we could allow subtitles so that we can see the OSD info?
Thanks.

**Comment by d4rken** [comment]
Would love to, but afaik there is currently no way to get the data, have you seen any other project manage to access it?

**Comment by terry-dactle** [comment]
Yeah fair enough, I had no idea, I thought it just grabbed the file emulating the SD card because it always makes it unavailable, I didn't realise that wasn't possible.

Thanks for your awesome work.

**Comment by d4rken** [comment]
The video data is output raw via USB, and then converted and displayed by the app.
It unfortunately doesn't contain any OSD data.
Some on fpv-out Discord are still looking it how to get OSD data, but no luck so far.
